go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/prometheus/client_golang v1.1.0
 	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
+	github.com/prometheus/common v0.6.0
```

Comment:
I don't know anything about prometheus releases, but should we try to get the 3 packages to all have the same version?

Reply:
`v0.6.0` is the latest for `Common` (https://github.com/prometheus/common/releases/tag/v0.6.0)

We could upgrade the other deps, but not sure what's different in them to move or not.

Reply:
Yeh, there doesn't seem to be any version numbering consistency across these packages 🤷‍♂